pyproject.toml

```diff
@@ -101,20 +101,67 @@ commit_message = "Version generated by python-semantic-release [ci skip]"
 [tool.poetry.plugins."pytest11"]
 fastapi-sqla = "fastapi_sqla._pytest_plugin"
 
-[tool.pylama]
-paths = "fastapi_sqla tests"
-linters = "pycodestyle,mccabe,pyflakes"
+[tool.ruff]
+target-version = "py39"
+
+[tool.ruff.lint]
+select = [
+  "E", # pycodestyle
+  "W", # pycodestyle warnings
+  "F", # pyflakes
+  "I", # isort
+  "N", # pep8-naming
+  "D", # pydocstyle
+  "UP", # pyupgrade
+  "ASYNC", # flake8-async
+  "S", # flake8-bandit
+  "B", # flake8-bugbear
+  "C4", # flake8-comprehensions
+  "SIM", # flake8-simplify
+  "TD", # flake8-todos
+  "ERA", # eradicate
+  "PL", # pylint
+  "RUF" # ruff
+]
+
+# Ignore some pydocstyle rules that Google convention enables
+ignore = [
+  "D100",
+  "D101",
+  "D102",
+  "D103",
+  "D104",
+  "D105",
+  "D106",
+  "D107",
+  "D212",
+  "D415",
+  "B008", # Enable using functions in default function args. Easier to work with FastAPI dependencies that way
+  "C408", # Enable using dict/list/tuple
+  "PLR0911", # Disable max number of return
+  "PLR0912", # Disable max number of branches
+  "PLR0913", # Disable max number of args
+  "PLR0915", # Disable max number of statements
+  "PLR2004", # Enable magic values
+  "SIM105", # Allow using try - except - pass
+  "TD001", # Enable FIXMEs
+  "TD002", # Enable TODOs without author
+  "TD003", # Enable TODOs without issue link
+  "N802", # FastAPI dependency conventions is title cased
+  "N803", # Same as N802
+  "N806", # Same as N802
+]
 
-[tool.pylama.linter.pycodestyle]
-# https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#line-length
-max_line_length = 88
+[tool.ruff.lint.extend-per-file-ignores]
```

Comment:
Standard Dialogue config + `S608` given the use of string construction in tests, with no security risks